tchannel/serializer/json.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import json
+
+from tchannel.schemes import JSON
+
+
+class JsonSerializer(object):
+    def __init__(self):
```

Comment:
@abhinav there's no need to use classes here at all - they should be pure functions